### PR TITLE
golang 1.9 and plugins updates.

### PR DESCRIPTION
**Pull request by jcvenegas**
Update golang to version 1.9.2 to avoid issues building   third party modules.

Update configuration from plugins.

### CODE REVIEW

File: jobs/virtcontainers-fedora-26-PR/config.xml [hunk 1]
```diff
@@ -92,6 +92,8 @@ set -e
 
 export ghprbPullId
 export ghprbTargetBranch
+export ghprbAuthorRepoGitUrl
```
Comment (chavafg) [inline jobs/virtcontainers-fedora-26-PR/config.xml:95]: Can you add a comment in the commit message that these changes were to comply with the virtcontainers CI changes? 
https://github.com/containers/virtcontainers/commit/1a638c0f173b9aa06d8e03943ae604655f1719b8
Or better, add a new commit.

**Comment by sboeuf** [comment]
@jcvenegas Once you've addressed @chavafg's comment, this LGTM

File: jobs/virtcontainers-fedora-26-PR/config.xml (hunk 1 above)
Comment (jcvenegas) [reply jobs/virtcontainers-fedora-26-PR/config.xml:95]: @chavafg , added new commit !

**Comment by jcvenegas** [comment]
@chavafg ping

**Comment by chavafg** [comment]
lgtm

**Pull request merged**